src/chapters/unit-testing/exercises.rst

```diff
@@ -0,0 +1,115 @@
+Exercises: Unit Testing
+=======================
+
+Work on these exercises in your local copy of the `csharp-web-dev-lsn5unittesting <https://github.com/LaunchCodeEducation/csharp-web-dev-lsn5unittesting>`_ repo.
+You were directed to fork and clone this repo earlier in the chapter. If you have not done so, do so now!
+Before starting work on the exercises, check out the ``exercises-start`` branch.
+
+.. admonition:: Tip
+
+   Earlier in the chapter, we instantiated an object of the ``Car`` class called ``test_car``.
+   Be sure to take note of the values of the ``make``, ``model``, ``GasTankSize``, and ``MilesPerGallon`` properties!
+   Having a note you can quickly reference of how big the gas tank is will you help you decide on values to use in your tests!
+
+``TestGasTankAfterDriving()``
+-----------------------------
+
+Add a test for the third TODO, "GasTankLevel is accurate after driving within tank range".
+
+#. Your test must use the ``Car`` method ``Drive()`` 
+
+   .. sourcecode:: csharp
+
+      test_car.Drive(50);
+
+#. With a value of ``50`` miles passed into ``Drive()``, we expect 
+   ``test_car`` to have a ``GasTankLevel`` of ``9``.
+
+   .. sourcecode:: csharp
+
+      Assert.AreEqual(9, test_car.GasTankLevel(), .001);
+
+``TestGasTankAfterExceedingTankRange()``
+----------------------------------------
+
+Add a test for the fourth TODO, "GasTankLevel is accurate after attempting to drive past tank range".
+
+#. You're on your own for this one. You'll need to simulate the ``Car``
+   travelling farther than it's ``GasTankLevel`` allows.
```

Comment:
farther than the ``gasTankLevel`` value allows.

Reply:
Done!